<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_head">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ключ ТМ:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Режим</string>
    </property>
    <addaction name="combat"/>
    <addaction name="training"/>
    <addaction name="combat_gen"/>
    <addaction name="training_gen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="past_key"/>
    <addaction name="copy_key"/>
    <addaction name="clear_key"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="dark_theme"/>
    <addaction name="light_theme"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="combat">
   <property name="text">
    <string>Боевой</string>
   </property>
   <property name="toolTip">
    <string>Ввод ключа, вывод КС.</string>
   </property>
  </action>
  <action name="training">
   <property name="text">
    <string>Тренировочный</string>
   </property>
   <property name="toolTip">
    <string>Ввод ключа, ввод КС, проверка КС.</string>
   </property>
  </action>
  <action name="combat_gen">
   <property name="text">
    <string>Боевой генерация</string>
   </property>
   <property name="toolTip">
    <string>Генерация ключа, вывод КС.</string>
   </property>
  </action>
  <action name="training_gen">
   <property name="text">
    <string>Тренировочный ген...</string>
   </property>
   <property name="toolTip">
    <string>Генерация ключа, ввод КС, проверка КС.</string>
   </property>
  </action>
  <action name="past_key">
   <property name="text">
    <string>Вставить ключ</string>
   </property>
   <property name="toolTip">
    <string>Вставить заранее скопированый  ключ.</string>
   </property>
  </action>
  <action name="clear_key">
   <property name="text">
    <string>Отчистить ключ</string>
   </property>
   <property name="toolTip">
    <string>Отчистить поле ввода ключа и КС.</string>
   </property>
  </action>
  <action name="copy_key">
   <property name="text">
    <string>Копировать ключ</string>
   </property>
   <property name="toolTip">
    <string>Копировать ключ в буфер обмена</string>
   </property>
  </action>
  <action name="dark_theme">
   <property name="text">
    <string>Темная</string>
   </property>
   <property name="toolTip">
    <string>Сменить тему на темную.</string>
   </property>
  </action>
  <action name="light_theme">
   <property name="text">
    <string>Светлая</string>
   </property>
   <property name="toolTip">
    <string>Сменить тему на светлую.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
